requirements-tests.txt

```diff
@@ -2,6 +2,4 @@ moto==1.3.13
 python-box~=3.4.0
 flake8~=3.7.7
 nose2~=0.9.1
-kubernetes~=10.0.1
-python-dotenv~=0.10.2
```

Comment:
Removed these because they are duplicated by `requirements.txt`; both of which are installed to run tests in the Makefile; making it easier to switch versions when running unit tests.